queue/main_test.go

```
@@ -6,62 +6,93 @@ import (
+		mockQueue := &MockQueue{}
 		assert.NotPanics(t, func() {
 			InitQueueConnection(mockQueue)
 		})
 	})
+
+	t.Run("should not panic when PublishMessage() returns error", func(t *testing.T) {
+		mockQueue := &QueueWrapper{Publish: func(exchange, key string, mandatory, immediate bool, msg amqp.Publishing) error {
+			return nil
+		}}
+		assert.NotPanics(t, func() {
+			mockQueue.PublishMessage([]byte("message"))
```

golangci-lint (1.64.8)

Error return value of `mockQueue.PublishMessage` is not checked

(errcheck)